client-dart-sdk/README.md

```diff
@@ -44,19 +41,51 @@ Create FeatureHub Repository that holds feature states by providing FeatureHub s
 ```dart
 repository = ClientFeatureRepository();
 
-featurehubApi = FeatureHubSimpleApi(
+fhConfig = FeatureHubConfig(
 'http://localhost:8903',
 [
 'default/806d0fe8-2842-4d17-9e1f-1c33eedc5f31/tnZHPUIKV9GPM4u0koKPk1yZ3aqZgKNI7b6CT76q'
 ],
 repository!);
 ```
 
-### 3. Request features state:
-Make the following call to request feature state updates: 
+### 3. Get features state from your code:
 
 ```dart
-featurehubApi!.request();
+class Sample extends StatelessWidget {
+  const Sample({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text('My app'),
+        ),
+        body: StreamBuilder<FeatureStateHolder>(
+            stream: repository!
+                .feature('COLOUR_VARIATION_FEATURE')
+                .featureUpdateStream,
+            builder: (context, snapshot) {
+              if (snapshot.hasData) {
+                return Container(
+                    color: snapshot.data?.booleanValue == true
+                        ? Colors.red
+                        : Colors.green,
+                    child: Text('Hello world!'));
+              } else {
```

Comment:
reverse the if!

if (!snapshot.hasData) return SizedBox.shrink();

return COntainer(....)

Reply:
why? it's the same thing?